esp-hal-smartled/Cargo.toml

```diff
@@ -14,25 +14,26 @@ targets  = ["riscv32imac-unknown-none-elf"]
 [dependencies]
 defmt             = { version = "1.0.1", optional = true }
 document-features = "0.2.10"
-esp-hal           = { version = "1.0.0-rc.0", features = ["unstable"] }
+esp-hal           = { version = "1.0.0-rc.0", features = ["requires-unstable"] }
 smart-leds-trait  = "0.3.1"
 
 [dev-dependencies]
 cfg-if = "1.0.0"
-esp-backtrace = { version = "0.16.0", features = [
+esp-backtrace = { version = "0.17.0", features = [
     "exception-handler",
     "panic-handler",
     "println",
 ] }
-esp-hal-embassy = "0.8.1"
+esp-hal-embassy = "0.9.0"
 embassy-executor = "0.7.0"
 embassy-time = "0.4.0"
-esp-println = "0.14.0"
+esp-println = "0.15.0"
 smart-leds = "0.4.0"
 
 [features]
 ## Implement `defmt::Format` on certain types.
 defmt = ["dep:defmt", "esp-hal/defmt"]
+unstable = ["esp-hal/unstable"]
```

Comment:
Users will already have esp-hal in their dependencies, they should set `unstable` on esp-hal directly. This feature is not required, and may even cause confusion (you can enable `unstable` on `esp-hal-smartled`, skip it on `esp-hal` and end up with unstable esp-hal features enabled anyway).